js.js

```diff
@@ -0,0 +1,91 @@
+  var li = $('#li');
+  var li1 = $('#li1');
+  var li2 = $('#li2');
+  var li3 = $('#li3');
+  var li4 = $('#li4');
+  var li5 = $('#li5');
+  var li6 = $('#li6');
+  var li7 = $('#li7');
+  var li8 = $('#li8');
+
+
+    li.on('mouseenter',show);
+    li1.on('mouseenter',show1);
+    li2.on('mouseenter',show2);
+    li3.on('mouseenter',show3);
+    li4.on('mouseenter',show4);
+    li5.on('mouseenter',show5);
+    li6.on('mouseenter',show6);
+    li7.on('mouseenter',show7);
+    li8.on('mouseenter',show8);

```

Comment:
Did you try to achieve the same result by using only one single class instead of 9? ;)